### PR TITLE
NOISSUE - Add version filter

### PR DESCRIPTION
**Pull request by 1998-felix**
### What does this do?
Add version filter to callhome sdk

### Which issue(s) does this PR fix/relate to?
Resolves ultravioletrs/issues#270

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No
### Notes

### CODE REVIEW

**Comment by drasko** [comment]
@SammyOina please review

**Comment by ianmuchyri** [comment]
@1998-felix  this PR has commits that have already been merged in the main branch I can see` Add city filter` and `Fix missing country in OpenAPI` have already been merged in the main branch. Check on that

**Review by ianmuchyri — APPROVED** [review]

File: timescale/timescale.go [hunk 1]
```diff
@@ -171,6 +171,11 @@ func generateQuery(filters callhome.TelemetryFilters) (string, map[string]interf
 		params["city"] = filters.City
 	}
 
+	if filters.Version != "" {
+		queries = append(queries, "version = :version")
```
Comment (SammyOina) [inline timescale/timescale.go:175]:
```suggestion
		queries = append(queries, "mf_version = :version")
```

**Review by SammyOina — CHANGES_REQUESTED** [review]

File: timescale/timescale.go (hunk 1 above)
Comment (SammyOina) [reply timescale/timescale.go]: you also have to manually test your PRs before submission

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**